include/inviwo/core/properties/fileproperty.h

```diff
@@ -82,7 +82,14 @@ class IVW_CORE_API FileProperty : public TemplateProperty<std::string> {
     virtual FileProperty* clone() const override;
     virtual ~FileProperty() = default;
 
-    virtual void set(const std::string& value) override;
+    /**
+     * Set the file name and also update the selected extension to the first one matching file.
```

Comment:
first one matching file? 
You mean?
Set file name and update the selected extension to that of the file.

Reply:
the file is just a string, and there can be may FileExtensions that match for example *.jpg (one per registered reader)